doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -323,6 +323,9 @@ Modem libraries
       The timestamp comes from the moment of scanning or neighbor measurements.
     * Removed dependency on the :ref:`lib_modem_jwt` library.
       The :ref:`lib_location` library now selects :kconfig:option:`CONFIG_NRF_CLOUD_REST_AUTOGEN_JWT` when using :kconfig:option:`CONFIG_NRF_CLOUD_REST`.
+    * Added a new feature, GNSS obstructed satellite visibility detection.
```

Comment:
"GNSS obstructed" sounds a bit weird. Would this work without GNSS? So just: "obstructed satellite visibility detection"

Reply:
@peknis How about "Added a new feature for GNSS, obstructed satellite visibility detection"?

Reply:
Updated as agreed offline.